### ISSUE TITLE
Error while parsing a valid VCF header

### ISSUE THREAD

**Issue by CastilloDel** [zaeleus/noodles#18]
If I try to parse the VCF example header from the [VCF specification](https://samtools.github.io/hts-specs/VCFv4.3.pdf) I get the following error.
Code:
```rust
  use noodles_vcf::Header;
  r#"##fileformat=VCFv4.3
##fileDate=20090805
##source=myImputationProgramV3.1
##reference=file:///seq/references/1000GenomesPilot-NCBI36.fasta
##contig=<ID=20,length=62435964,assembly=B36,md5=f126cdf8a6e0c7f379d618ff66beb2da,species="Homo sapiens",taxonomy=x>
##phasing=partial
##INFO=<ID=NS,Number=1,Type=Integer,Description="Number of Samples With Data">
##INFO=<ID=DP,Number=1,Type=Integer,Description="Total Depth">
##INFO=<ID=AF,Number=A,Type=Float,Description="Allele Frequency">
##INFO=<ID=AA,Number=1,Type=String,Description="Ancestral Allele">
##INFO=<ID=DB,Number=0,Type=Flag,Description="dbSNP membership, build 129">
##INFO=<ID=H2,Number=0,Type=Flag,Description="HapMap2 membership">
##FILTER=<ID=q10,Description="Quality below 10">
##FILTER=<ID=s50,Description="Less than 50% of samples have data">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##FORMAT=<ID=GQ,Number=1,Type=Integer,Description="Genotype Quality">
##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Read Depth">
##FORMAT=<ID=HQ,Number=2,Type=Integer,Description="Haplotype Quality">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NA00001	NA00002	NA00003"#
.parse::<Header>().unwrap();

```
Output:
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: InvalidContig(InvalidRecord)'
```

The error disappears when I remove the `md5=f126cdf8a6e0c7f379d618ff66beb2da,` part in the contig field.

**Comment by zaeleus** [comment]
Thanks for the report, @CastilloDel!

This is fixed in fb8bfe353b7b8e5979b78d82cbfe8e95e5077093.

**Comment by CastilloDel** [comment]
That was so quick! Thanks!
